javav2/example_code/kms/src/test/java/AmazonKMSTest.java

```diff
@@ -75,6 +75,7 @@ public void HelloKSM() {
     @Tag("IntegrationTest")
     @Order(2)
     public void TestScenario() {
+      /*

```

Comment:
If a test is commented out that's not a very helpful test. Uncomment and fix.

Reply:
Nice comment - meant to do that.